osu.Game/Graphics/Containers/LinkFlowContainer.cs

```diff
@@ -57,7 +57,7 @@ public void AddLinks(string text, List<Link> links)
             createLink(AddText(text, creationParameters), new LinkDetails(LinkAction.External, url), url);
 
         public void AddLink(string text, Action action, string tooltipText = null, Action<SpriteText> creationParameters = null)
-            => createLink(AddText(text, creationParameters), new LinkDetails(LinkAction.Custom, null), tooltipText, action);
+            => createLink(AddText(text, creationParameters), new LinkDetails(LinkAction.Custom, @"action"), tooltipText, action);
```

Comment:
Could alternatively use `string.Empty` here, maybe, for a slightly less magic value?

Reply:
Yeah, almost changed this myself after noticing how other use cases use the value. Will update.